templates/repo/blame.tmpl

```diff
@@ -55,7 +55,11 @@
 								<span id="L{{$row.RowNumber}}" data-line-number="{{$row.RowNumber}}"></span>
 							</td>
 							{{if $.EscapeStatus.Escaped}}
-								<td class="lines-escape">{{if $row.EscapeStatus.Escaped}}<a href="" class="toggle-escape-button" title="{{$.locale.Tr "repo.line_unicode"}}"></a>{{end}}</td>
+								<td class="lines-escape">
+									{{if $row.EscapeStatus.Escaped}}
+										<a href="" class="toggle-escape-button" title="{{if $row.EscapeStatus.HasInvisible}}{{$.locale.Tr "repo.invisible_runes_line"}} {{end}}{{if $row.EscapeStatus.HasAmbiguous}}{{$.locale.Tr "repo.ambiguous_runes_line"}}{{end}}"></a>
```

Comment:
Could you check whether `class="tooltip"` with `data-content` works on these? I think we should generally prefer these over `title` attributes. Thought there might be CSS stacking context issues until https://github.com/go-gitea/gitea/pull/20428 is done.

Reply:
The code here isn't changing the structure of the DOM so if `class="tooltip" data-content="..."` works on the current blame page then it would work here.

Given the tooltip works on the date portion of the blame table I would guess it would work.

I guess the question is whether to make the change here or not. Given you've mentioned it I guess I need to do it.

Reply:
It actually looks rather weird due to overflowing on to a second line:

![Screenshot from 2022-07-22 09-52-38](https://user-images.githubusercontent.com/1824502/180402594-32f8db20-925c-46fe-b4cf-4bec22ae083e.png)

Reply:
So I think we might want to consider fixing that in a different PR - possibly along with the other places that are using `title=""` instead of `tooltip`.

Actually might it be better to simply have a bit of JS that handles the title elements and tooltipises them for us? That way we could actually drop the class="tooltip" and move back to plain old html? Or would that be too slow?

Reply:
Overlow issue is fomantic related, it won't happen after https://github.com/go-gitea/gitea/pull/20428.

Agree we can migrate `title` to tooltips later. Yes, JS to replace title with tooltip may be possible, but I'd rather explicitly migrate them to tooltips and test individually.